pkg/client/cache/index.go

```diff
@@ -24,7 +24,11 @@ func ImageStreamReferenceIndexFunc(obj interface{}) ([]string, error) {
 			}
 			params := trigger.ImageChangeParams
 			name, _, _ := imageapi.SplitImageStreamTag(params.From.Name)
-			keys = append(keys, params.From.Namespace+"/"+name)
+			namespace := params.From.Namespace
```

Comment:
@deads2k last thing that needs review. On master we currently default empty namespaces for ICT references to the config namespace but this is not true for older versions so our index needs to take empty namespace into account and use the config ns.